examples/zkapps/04-zkapp-browser-ui/ui/app/zkappWorker.ts

```diff
@@ -0,0 +1,54 @@
+import { Mina, PublicKey, fetchAccount } from 'o1js';
+import * as Comlink from "comlink";
+import type { Add } from '../../contracts/src/Add';
+
+type Transaction = Awaited<ReturnType<typeof Mina.transaction>>;
+
+const state = {
+  Add: null as null | typeof Add,
```

Comment:
nit: maybe calling this `AddContract` and renaming `zkapp` to `contractInstance` makes this a bit clearer

Reply:
That's a good idea.